pysph/sph/wc/edac.py

```diff
@@ -473,7 +473,8 @@ def stage2(self, d_idx, d_x0, d_y0, d_z0, d_x, d_y, d_z,
 class EDACScheme(Scheme):
```

Comment:
You seem to have changed the permissions of the file to be executable, can you please change this back to 644?

Reply:
done